libs/localization/assets/locales/en/char_Kirara.json

```diff
@@ -0,0 +1,6 @@
+{
+  "maxShield": "Max DMG Absorption",
```

Comment:
Is this a common string?

Reply:
It's not common because there are only two characters who can stack shield. But the item should be correct. See [Thoma](https://github.com/frzyc/genshin-optimizer/blob/master/libs/localization/assets/locales/en/char_Thoma.json) (except that's maximum but not max).
```
  "maxShield": "Maximum DMG Absorption",
  "maxPyroShield": "Maximum Pyro DMG Absorption"
```

The in-game text is 'Max Shield DMG Absorption'. So if you want to be as accurate as possible, both should be changed.